Tools/peg_generator/pegen/build.py

```diff
@@ -21,6 +22,12 @@
 MOD_DIR = pathlib.Path(__file__).parent
 
 
+def get_extra_flags(compiler_flags, compiler_py_flags_nodist):
+    flags = sysconfig.get_config_var(compiler_flags)
+    py_flags_nodist = sysconfig.get_config_var(compiler_py_flags_nodist)
+    return (flags + ' ' + py_flags_nodist).split()
```

Comment:
What about an fstring here to avoid the need for parens?

```suggestion
    return f'{flags} {py_flags_nodist}'.split()
```